data/events/2022-tampa.yml

```
@@ -21,27 +21,27 @@ cfp_open: "false"
 #cfp_link: "https://www.papercall.io/dodtampabay" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
 cfp_link: "https://www.papercall.io/tampadod"
 
-registration_date_start: "" # start accepting registration. Leave blank if registration is not open yet
-registration_date_end: "" # close registration. Leave blank if registration is not open yet.
+registration_date_start: 2022-08-14T00:00:00-05:00 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: 2022-08-21T23:59:59-04:00 # close registration. Leave blank if registration is not open yet.
```

I think this should be October, not August, for your rego close date (presuming you want rego open for more than 7 days total)
```suggestion
registration_date_end: 2022-10-21T23:59:59-04:00 # close registration. Leave blank if registration is not open yet.
```

---

Great catch and yes it is supped to be October 21st.